guide/12-enrich-data-with-thematic-information/part7_discover_and_enrich_standard_geographies.ipynb

```diff
@@ -0,0 +1,1165 @@
+{
```

Comment:
<p>Variation can now be calculated for the variables to identify those with exceedingly high variance. Analysis can be used for feature selection, feature reduction, addressing covariance between variables, and performing modeling.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1405/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>